var/spack/repos/builtin/packages/snappy/package.py

```diff
@@ -10,9 +10,10 @@ class Snappy(CMakePackage):
     """A fast compressor/decompressor: https://code.google.com/p/snappy"""
 
     homepage = "https://github.com/google/snappy"
-    url      = "https://github.com/google/snappy/archive/1.1.7.tar.gz"
+    url      = "https://github.com/google/snappy/archive/1.1.8.tar.gz"
 
     version('1.1.7', sha256='3dfa02e873ff51a11ee02b9ca391807f0c8ea0529a4924afa645fbf97163f9d4')
+    version('1.1.8', sha256='16b677f07832a612b0836178db7f374e414f94657c138e6993cbfc5dcc58651f')
```

Comment:
Can you please put this version before `1.1.7`? Asking since all the packages in the builtin repository declare version in descending order (newer first).

Reply:
done